noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1364,9 +1369,10 @@
             </info>
             <warnings>
                 <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, d.systemid document_description_system_id, dokumentstatus document_status
+                    SELECT r._ref_arkivdel series_system_id, f.mappeid file_id, r.registreringsid record_id, d.systemid document_description_system_id, dokumentstatus document_status
                     FROM arkivstruktur.dokumentbeskrivelse d
                     JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
```

Comment:
At a first glance, it seems OK to use `_id` and `_parent_id` in this join. However, by keeping the join with system IDs we can represent more accurately what was found in the extraction package. When using the internal `_id` and `_parent_id` fields we risk ignoring some issues such as duplicate system IDs, for example.